### ISSUE TITLE
Sort out SQLite installer for winrt

### ISSUE THREAD

**Issue by slodge** [MvvmCross/MvvmCross#307]
@dsplaisted says this will help http://nuget.org/packages/microsoft.bcl.compression

**Comment by martijn00** [comment]
We use Akavache in our WinRT app, and that uses SQLite. So this seems fixed to me...

**Comment by paulcbetts** [comment]
Yeah, while having native DLL support in NuGet would be Nice(tm), SQLitePCL_Raw is all you need right now (and in general, you can use targets / props files to make native DLLs happen).

**Comment by slodge** [comment]
I'd be very happy to see this closed... but I'm not sure this is fixed... 

This is about the Mvx SQLite .nuspec files. Because WinRT is multi-platform, we should install separate Mvx.Sqlite.dll assemblies for each ARM/x86/x64 platform. There's no difference in code, but there is a difference in compilation.

If we could fix it by switching the Mvx SQLite code to use _Raw instead (so it becomes platform-agnostic) then I'd be happy to see this closed :)
